test/CMakeLists.txt

```diff
@@ -241,9 +241,9 @@ if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "^(Apple)?Clang$")
     target_compile_options(unit-test PRIVATE -Wno-undef -Wno-reserved-id-macro)
 endif()
 
-# So many type_prop tests these days that we need to set /bigobj flag for MSVS.
+# So many type_prop tests these days that we need to set /bigobj flag for MSVC.
```

Comment:
Is this specifically the type_prop test file? This file is being broken into individual files, one for each op, so we might be able to not set /bigobj. Is there any downside to setting this? If no downside then it really does not matter.

Reply:
I put MSVC just to eliminate a custom MSVS option because I use NMake instead of  Visual Studio. I have no idea about disadvantages of `/bigobj`, sorry.